components/remote-environment-broker/internal/mapping/controller.go

```diff
@@ -139,7 +152,7 @@ func (c *Controller) processNextItem() bool {
 	case err == nil:
 		c.queue.Forget(key)
 
-	case isTemporaryError(err) && c.queue.NumRequeues(key) < maxEnvironmentMappingProcessRetries:
+	case c.queue.NumRequeues(key) < maxEnvironmentMappingProcessRetries:
```

Comment:
handling temporary errors introduce a lot of difficulties - so we decided to simplify it and on every error retries can be performed.